chapters/analysis.typ

```diff
@@ -41,8 +41,12 @@ The "Abbreviate Function Template" refactoring (@abbreviate_function_template),
 in which case it must not be applied.
 It also should be considered that in some cases, not the whole code can be analyzed or is not even available.
 
-// COR Zusätzliche Schwierigkeit in C++: Präprozessor, womit quasi fast jeder Name umdefiniert werden kann. 
-// VINA No idea how to document this...
+The preprocesser capabilites of C++ pose another challenge to refactoring operations.
```

Comment:
Hesch irgendwelchi infos nahgluegt ufere websiite?

Reply:
eh nö, has eifach gschrebe

Reply:
ich glaub da müsst mer scho na e quelle dezue finde

Reply:
han eifach d'referenz zum preprocessor ine to jetz, han suscht nüt gfunde